cmd/describe/ingress/cmd.go

```diff
@@ -0,0 +1,185 @@
+package ingress
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/helper"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/output"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+// Regular expression to used to make sure that the identifier given by the
+// user is safe and that it there is no risk of SQL injection:
+var ingressKeyRE = regexp.MustCompile(`^[a-z0-9]{3,5}$`)
+
+var Cmd = &cobra.Command{
+	Use:     "ingress",
+	Short:   "Show details of the specified ingress within cluster",
+	Example: `rosa describe ingress <ingress_id> -c mycluster`,
+	Run:     run,
+	Args: func(_ *cobra.Command, argv []string) error {
+		if len(argv) != 1 {
+			return fmt.Errorf(
+				"Expected exactly one command line parameter containing the id of the ingress",
+			)
+		}
+		return nil
+	},
+}
+
+func init() {
+	ocm.AddClusterFlag(Cmd)
+	output.AddFlag(Cmd)
+}
+
+func run(_ *cobra.Command, argv []string) {
```

Comment:
@gdbranco I've been trying to encourage folks to delegate to another function so that the main work of their command can be tested. At the minute with the `os.Exit()` calls. We can't test this. 

Recent examples are the `config get` and `config set` commands added by Tirth. 

It's also something I'm working to make easier in #1837